content-src/components/Spotlight/Spotlight.scss

```diff
@@ -122,3 +122,10 @@
     margin: 0;
   }
 }
+
+// overrides for Top Sites styling experiment
```

Comment:
It'd be nice if the comment gave some context to why site-icon is being turned off here.